From 5e9627e661edbf534d0649153df3dddcca7c47f2 Mon Sep 17 00:00:00 2001
From: Randy Li <randy.li@rock-chips.com>
Date: Thu, 24 Nov 2016 10:23:17 +0800
Subject: [PATCH] ARM: dts: rockchip: switch to vpu-service in asus miniarm

We decide to offer the video support in old way.

Change-Id: I0b30f7b2b559596ee124b8558cd54e0ff090ab79
Signed-off-by: Randy Li <randy.li@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-miniarm.dts | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3288-miniarm.dts b/arch/arm/boot/dts/rk3288-miniarm.dts
index 5bd046c2febb..f58d9dcb0834 100644
--- a/arch/arm/boot/dts/rk3288-miniarm.dts
+++ b/arch/arm/boot/dts/rk3288-miniarm.dts
@@ -466,7 +466,11 @@
 	status = "okay";
 };
 
-&vpu {
+&hevc_service {
+	status = "okay";
+};
+
+&vpu_service {
 	status = "okay";
 };
 
-- 
2.35.3

